Makefile

```diff
@@ -42,10 +42,11 @@ FUSES      = -U hfuse:w:0xd8:m -U lfuse:w:0xff:m
 # Tune the lines below only if you know what you are doing:
 
 AVRDUDE = avrdude  $(PROGRAMMER) -p $(DEVICE) -B 10 -F -D
-COMPILE = avr-gcc -Wall -Os -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I. -ffunction-sections
+COMPILE = avr-gcc  -Wall -Os -g -DF_CPU=$(CLOCK) -mmcu=$(DEVICE) -I. -ffunction-sections
```

Comment:
I don't really believe this should be an issue, but the motor was making a different sound Friday afternoon. This issue is more likely due to missed deadlines by the microprocessor as opposed to using too much space, but I will try without the -g next week.